app/javascript/components/mappers/domainToApi/lease.js

```diff
@@ -0,0 +1,14 @@
+import { createFieldMapper } from '~/utils/objectUtils'
+import { currencyToFloat } from '~/utils/utils'
+
+// Lease is sent to backend with domain keys.
+export const leaseFieldMapper = {
```

Comment:
and this one

Reply:
@akegan with this and the above, it is nice to see how we can use what we already have to accomplish the goal of this story. i also wonder if it may be worth adding a bit more new work to achieve greater clarity, by doing value conversion/formatting separately from field mapping. what are your thoughts on this?

Reply:
do you have an idea of where it might make more sense to do the conversion/formatting?

Reply:
update on this conversation: @akegan and I discussed on a call and decided to wait for the upcoming upgrade to a new form library to break out these kinds of formatting.